src/App.js

```diff
@@ -1,18 +1,22 @@
-import "./App.css";
-import Header from "./components/Header/Header";
-import Main from "./components/Main/Main";
-import Section from "./components/Section/Section";
-import Footer from "./components/Footer/Footer";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Shared from "./pages/Shared";
+import HomePage from "./pages/HomePage";
+import Folder from "./pages/Folder";
+import FolderEmpty from "./pages/FolderEmpty";
 
-function App() {
+function Main() {
   return (
-    <>
-      <Header />
-      <Main />
-      <Section />
-      <Footer />
-    </>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/">
+          <Route index element={<HomePage />} />
+          <Route path="shared" element={<Shared />} />
+          <Route path="folder" element={<Folder />} />
+          <Route path="folderEmpty" element={<FolderEmpty />} />
+        </Route>
```

Comment:
```suggestion
          <Route path="/" element={<HomePage />} />
          <Route path="/shared" element={<Shared />} />
          <Route path="/folder" element={<Folder />} />
          <Route path="/folder-empty" element={<FolderEmpty />} />
```
[중요도: 낮음] 이런식으로 하셔도 되었었을텐데... <Route path="/">로 묶은 이유가 특별히 있을까요?
[중요도: 높음] url pahtname의 경우는 웬만해선 kebab-case가 좋습니다. SEO관점에서도 좋고 가독성에서도 좋습니다.
https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/Why-you-should-make-kebab-case-a-URL-naming-convention-best-practice
위의 게시글 참고해주세요~

Reply:
코드잇 강의에서 저렇게 /를 위로 올리고 메인에다 index라고 해주면 
따로 경로설정을 안해줘도 된다고 해서 저렇게 했는데
멘토님이 말씀해주신 방법이 좀 더 간결하고 좋은것 같습니다 수정하겠습니다!